docs/loadtest/README.md

```diff
@@ -5,36 +5,23 @@ This tool is used to generate continues load on the Eventing components.
 It does that by sending Cloudevents to the Eventing publisher proxy forever and consume them inside Kyma functions.
 Ideally, it should be used when JetStream is used as the active Eventing backend.
 
-## Eventing infrastructure
+## Subscriptions
 
-For testing against **Legacy Events** set `use_legacy events` to `true`. Leave it to `false` to test against **Cloud Events**.
+The publisher watches all subscriptions in the cluster.
+To activate sending events for one subscription you have to label that subscription with one of these labels:
+- eventing-loadtest: legacy 
+- eventing-loadtest: cloudevent
 
-The load test will create:
+The load tester will then create events using the event types configured in the subscription and send them with the format configured in the label.
 
-1. **Two** Kyma function subscribers.
-2. **Two** Kyma subscriptions.
+The load generated per event type will be extracted from the event type. To do this the event type must match the following pattern:
+`<event-name>.v<Number of events per second>`, eg. `order.created.v500` will create events of type order.create.v500 and the sender will try to publish 500 events per second.
 
-| Namespace      | Subscription   | Subscriber            |
-|----------------|----------------|-----------------------|
-| eventing-test  | subscription-0 | loadtest-subscriber-0 |
-| eventing-test  | subscription-1 | loadtest-subscriber-1 |
-
-> Note: Each Kyma subscription is configured with `N` unique event types.
-
-## Event types naming conventions
-
-1. **There are two event names**:
-    - The `subscription-0` has `order.created`.
-    - The `subscription-1` has `order.updated`.
-
-2. **Events carry the `eps` information**:
- - EPS **50**: `order.created.v0050`.
- - EPS **90**: `order.created.v0090
+> Note: Encoding the EPS in the event type version is used only for debugging purposes and is not a production use-case.
 
-*Cloud Events* have the `eps` encoded at the end as part of the `version`.
-*Legacy Events* have the `eps`  body in the `event-type` field of the body.
+> There is no special handling of subscriptions otherwise. The `sink` and also the `maxInFlight` settings will be respected.
 
-> Note: Encoding the EPS in the event type version is used only for debugging purposes and is not a production use-case.
+This publisher works best in combination with the loadtest-subscriber. Two instances of the subscriber are deployed using the kustomize deployment. Configure the subscriptions to use those by setting the sink to `loadtest-subscriber-0.eventing-test.svc.cluster.local` or `loadtest-subscriber-1.eventing-test.svc.cluster.local`
```

Comment:
Do we still need to have 2 sinks? Does it make sense to always use the same sink and delete the other?

Reply:
the two sinks are configured to:
- 1. run on the same node as ec
- 2. run on different node than ec

but i agree... the whole concept of those sinks should be reconsidered